File: cipher.cpp
#include <iostream>
#include <string>
using namespace std;

string decrypt(const string& text, int shift = 3) {
    string result = "";
    for (char ch : text) {
        if (isalpha(ch)) {
            char offset = isupper(ch) ? 'A' : 'a';
            result += char((ch - offset - shift + 26) % 26 + offset);
        } else {
            result += ch;
        }
    }
    return result;
}

int main() {
    string text = "ufvf{hlgd_nrqr_nrwkd}";
    string decryptedText = decrypt(text);
    cout << "Decrypted Text: " << decryptedText << endl;
    return 0;
}

File: cipher.py
def decrypt(text, shift=3):
    result = ""
    for ch in text:
        if ch.isalpha():
            offset = ord('A') if ch.isupper() else ord('a')
            result += chr((ord(ch) - offset - shift + 26) % 26 + offset)
        else:
            result += ch 
    return result


text = "ufvf{hlgd_nrqr_nrwkd}"
decrypted_text = decrypt(text)
print("Decrypted Text:", decrypted_text)
